src/components/AuthButtons/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useAuth0 } from '@auth0/auth0-react';
+
+import Button from '@mui/material/Button';
+
+const LoginButton = () => {
+    const { loginWithRedirect } = useAuth0();
+
+    return (
+        <Button variant="contained" type="button" onClick={() => loginWithRedirect()}>
+            Log in
+        </Button>
+    );
+};
+
+export const SignupButton = () => {
+    const { loginWithRedirect } = useAuth0();

```

Comment:
Al presionar el botón me sale un error `404 not found`. Analizando el tema, hay un `/` que genera el error en la url de redirect. Si la quito manualmente funciona (ver imagen):
![2022-01-21_13-17-55](https://user-images.githubusercontent.com/8030746/151234096-d71f4cdd-ccf0-4a6a-8111-a3c789a8a8c4.jpg)